### ISSUE TITLE
Allow third-party font formulas

### ISSUE THREAD

**Issue by ronaldtse** [fontist/formulas#75]
Some people may want to install fonts from private locations or do not match the "publicly available font" criteria we have in the official Fontist formula repository.

Perhaps we should allow people to build and use their own Fontist formula repositories.

**Comment by alemorozov** [comment]
This was resolved in https://github.com/fontist/fontist/issues/238 and documented [here](https://github.com/fontist/fontist#authoring-fontist-formulas).

Let's close it? Or there are some thoughts on this issue?

**Comment by ronaldtse** [comment]
I think we wanted to deal with non-private (i.e. public) formula repos. Could we make one to try out?

**Comment by alemorozov** [comment]
Ah, understood: public formula repos but not the main (our) repo. Ok, let's try out.

**Comment by alemorozov** [comment]
Had an issue when a third party repo has a non-default branch name. Fixed it in #268.

Created [a public repo](https://github.com/fontist/example-formulas) to test.

```
bin/fontist repo setup https://github.com/fontist/example-formulas.git
bin/fontist install -f "lucida grande"
bin/fontist update
```

They all work.

**Comment by ronaldtse** [comment]
Closing, thanks @alexeymorozov !
